jazz-build-module/environment-deployment-metadata-loader.groovy

```diff
@@ -0,0 +1,499 @@
+#!groovy?
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonOutput
+import groovy.transform.Field
+@Field def Util
+
+echo "Environment and deployment metadata module loaded successfully"
+
+/**
+ * The environment-deployment loader module. This module will load all environment and deployment related  metadata from respective  tables or Apis 
+ * get it ready for Jenkins builds. It loads data for all service types. 
+*/
+
+
+@Field def g_login_token
+@Field def service_config
+@Field def config_loader
+@Field def scm_module
+@Field def g_service_branch 
+@Field def g_git_commit_hash
+@Field def g_git_commit_url
```

Comment:
rename this global variable to g_git_repo_url

Reply:
Renamed